.ldrelease/update-bitbucket-metadata.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
 
 set -ex
-CR_RELEASE_VERSION=${1:-$LD_RELEASE_VERSION}
+# Read from the command line so we can debug this script. Defaults to releaser env variable.
+RELEASE_VERSION=${1:-$LD_RELEASE_VERSION}
 
-README_BB=build/metadata/bitbucket/README.md
-README_BB_TEMP=${README_BB}.tmp
-sed "s#- pipe: launchdarkly/ld-find-code-refs-pipe.*#- pipe: launchdarkly/ld-find-code-refs-pipe:${CR_RELEASE_VERSION}#g" ${README_BB} > ${README_BB_TEMP}
-mv ${README_BB_TEMP} ${README_BB}
+README=build/metadata/bitbucket/README.md
+README_TEMP=${README}.tmp
+sed "s#- pipe: launchdarkly/ld-find-code-refs-pipe.*#- pipe: launchdarkly/ld-find-code-refs-pipe:${RELEASE_VERSION}#g" ${README} > ${README_TEMP}
+mv ${README_TEMP} ${README}
 
-VERSION_PIPE_CONTAINER=build/metadata/bitbucket/pipe.yml
-VERSION_PIPE_CONTAINER_TEMP=${VERSION_PIPE_CONTAINER}.tmp
-sed "s#image: launchdarkly/ld-find-code-refs-bitbucket-pipeline:.*#image: launchdarkly/ld-find-code-refs-bitbucket-pipeline:${CR_RELEASE_VERSION}#g" ${VERSION_PIPE_CONTAINER} > ${VERSION_PIPE_CONTAINER_TEMP}
-mv ${VERSION_PIPE_CONTAINER_TEMP} ${VERSION_PIPE_CONTAINER}
```

Comment:
I replaced all BB/GH specific references here in the hope that in the future we can somehow re-use a single script to update both repos.